tools/fake_ide.ml

```diff
@@ -252,11 +252,9 @@ let eval_print l coq =
       let to_id, _ = get_id id in
       eval_call (query (0,(phrase, to_id))) coq
   | [ Tok(_,"WAIT") ] ->
-      let phrase = "Stm Wait." in
-      eval_call (query (0,(phrase,tip_id()))) coq
+      eval_call (wait ()) coq
   | [ Tok(_,"JOIN") ] ->
-      let phrase = "Stm JoinDocument." in
-      eval_call (query (0,(phrase,tip_id()))) coq
+      eval_call (status true) coq
```

Comment:
Everything seems obvious except this part of the patch. Is it documented somewhere that `Status` is equivalent to `Join`?

Reply:
It certainly needs some fixing here: https://github.com/coq/coq/blob/master/dev/doc/xml-protocol.md#command-status.

Reply:
Does it look ok, now?

Reply:
Note that my question was not a rhetorical way of asking for some documentation. It was a genuine interrogation, since, if your change was correct, it would beg the question of why the protocol originally contained some duplication.

Reply:
Well, running navigation commands in queries anyway seems like a hack. What we should do is have a static classification (i.e. a dedicated type) for vernacular queries, and a parser for them. Currently, nothing prevents you from writing `Undo` in the query pannel of CoqIDE, so that's supported by the protocol but should eventually go away IMHO. And the duplication would be then resolved.